### DISCUSSION TITLE
Validation with multiple validation errors

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#4407] [Q&A] [answered]
Hello,

I'm currently migrating a complex validation framework to the build in validation and I'm searching for the possiblity to display multiple errors to one field. There seems to be the possibility to use "Multiline" inside the "ValidationError", but I can not find any example. Can somebody show me, how multiline is supposed to be used?
In my example we can get each error as `IEnumerable<String>`

**Comment by stsrki** [comment, marked as answer]
First, the `Multiline` need to be enabled on `ValidationError`. Next, that multi-lined messages will only work with **data annotations** and **fluent validation** validators. Regular Validator handler doesn't support multiline messages. Technically it should work with some minor changes to the codebase, but we would need to decide on the API.

**Reply by NeoCoderMatrix86** [reply]
Ok, what is **fluent validation**? Is there a possiblity to set a multiline error by code?

**Reply by stsrki** [reply]
You can see fluent validation at https://docs.fluentvalidation.net/en/latest/. And the way it is used in Blazorise https://blazorise.com/docs/extensions/fluent-validation

**Reply by NeoCoderMatrix86** [reply]
Ok, I think I get it now. One question: Is there a possibility to use fluent validation in combination with an AutoComplete (https://blazorise.com/docs/extensions/autocomplete) component? I only see custom validators?!

**Reply by stsrki** [reply]
It's not possible at the moment. Our AutoComplete is a _different beast_ from all other components so we had to figure a best way to add support for different validation modes.
